### ISSUE TITLE
Renaming HID device

### ISSUE THREAD

**Issue by heqsde** [fasteddy516/CircuitPython_JoystickXL#25]
Hey, right now is possible to change the name of your HID device, instead of the old standard CircuitPython HID.

Per #[https://github.com/adafruit/circuitpython/pull/8989]

Cheers.

**Comment by Mick3DIY** [comment]
Yes, more infos "Notable changes to 9.0.0 since 8.x.x" in https://github.com/adafruit/circuitpython/releases/tag/9.0.0
